layer/tile/Bing.js

```diff
@@ -1,131 +1,169 @@
+// Bing maps API: https://docs.microsoft.com/en-us/bingmaps/rest-services/
+
 L.BingLayer = L.TileLayer.extend({
 	options: {
-		subdomains: [0, 1, 2, 3],
-		type: 'Aerial',
+		// imagerySet: https://docs.microsoft.com/en-us/bingmaps/rest-services/imagery/get-imagery-metadata#template-parameters
+		// supported:
+		// - Aerial, AerialWithLabels (Deprecated), AerialWithLabelsOnDemand
+		// - Road (Deprecated), RoadOnDemand
+		// - CanvasDark, CanvasLight, CanvasGray
+		// not supported: Birdseye*, Streetside
+		type: 'RoadOnDemand',
```

Comment:
default type should be `Aerial` in order to prevent changes on existings maps

Reply:
I believe that 'RoadOnDemand' is better choice for default.
Besides, it's unlikely that even existing maps update themselves automatically, and every maintainer easily fix this option if needed.
So I propose consider this like major version change (and we just talked about repo change)

Reply:
> I believe that 'RoadOnDemand' is better choice for default.

I believe that it's nicer for our users who simply do `L.BingLayer(apiKey)` that their map doesn't switch layer.

> and every maintainer easily fix this option if needed

and more easy to do nothing :)

Reply:
> and more easy to do nothing :)

Yeah, why update if all working just well in old good version?)
Ok, let it stay 'Aerial' here, but change when transfer to separate repo?